ui/src/App.js

```diff
@@ -398,13 +399,13 @@ class App extends Component {
   /**
    * Converts an Array of filter strings back to a Map of filters
    * Example:
-   * In: ["Gender=female", "Gender=male", "Population=American"]
+   * In: ["Gender%3Dfemale", "Gender%3Dmale", "Population%3DAmerican"]
    * Out: {"Gender" => ["male", "female"], "Population" => ["American"]}
    */
   filterArrayToMap(filterArray) {
     let filterMap = new Map();
     filterArray.forEach(function(pair) {
-      pair = pair.split("=");
+      pair = pair.split("%3D");
```

Comment:
Instead of hardcoding %3D, how about using `encodeURIComponent('=')` instead? I can never remember what these strings actually are. 

(Same with the %7C strings below)

Reply:
Good idea, done